web/src/frontend/classic/stores/Documentation.ts

```diff
@@ -0,0 +1,80 @@
+import { writable } from 'svelte/store';
+
+export type DocCategory = {
+    name: string;
+    description: HTMLElement;
+    sections: DocSection[];
+}
+export type DocSection = {
+    name: string,
+    content: HTMLElement
+}
+
+export const documentation = writable<DocCategory[]>([]);
```

Comment:
May be preferred to use a readable store, so it can only be assigned internally and not by any other components.

Reply:
Oh right. It was only writable first because i did some other things like storing collapses. Forgot to switch to readable when i removed it.